content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+  field. The number should be in a `XX,XXX.XX` format.
+- :guilabel:`Proposed Salary - Extra advantages...`: If any extra advantages are beng offered to the
+  applicant, enter it in this field. This should be short and descriptive, such as `1 week extra
+  vacation` or `dental plan`.
+- :guilabel:`Availability`: Using the calendar module in the drop-down menu, select the available
+  start date for the applicant.
+
+Application summary tab
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Any additional details or notes that should be added to the applicant's card can be typed into this
+field.
+
+.. image:: recruitment/new-applicant.png
+   :align: center
+   :alt: All the fields for a new applicant form entered.
+
+
+First interview
+---------------
+
+After an applicant has passed the :guilabel:`Initial Qualification` stage, they next move to the
+:guilabel:`First Interview` stage. To move the applicant to the next stage, drag and drop the
+applicant's card to the :guilabel:`First Interview` stage, or click on the :guilabel:`First
+Interview` stage button at the top of the individual applicant's card.
+
+When the applicant's card moves to the :guilabel:`First Interview` stage, an acknowledgement email
+is automatically sent to the applicant. The email is created using an email template titled
+:guilabel:`Applicant: Acknowledgement`. This email template is pre-configured in Odoo, and contains
+dynamic placeholders for the job position, the recruiter's name, and the company's information. Once
+the email is sent, the email appears in the chatter.
+
+.. image:: recruitment/applicant-acknowledgement.png
+   :align: center
+   :alt: An email is automatically sent once an applicant is moved to the First Interview stage. The
+         email is seen in the chatter, and is sent using a template.
+
+.. _recruitment/schedule:
+
+Schedule a meeting
+~~~~~~~~~~~~~~~~~~
+
+When a date and time have been agreed upon for both the applicant and interviewer, the interview can
+be scheduled. To schedule an interview, whether a phone or in-person interview, click the
+:guilabel:`Meeting` smart button at the top of the applicant's record.
+
+.. note::
+   The meeting smart button will display :guilabel:`No Meeting` if no meetings are currently
+   scheduled. For new applicants who are new to the :guilabel:`First Interview` stage, this will be
+   the default. If there are any meetings already scheduled, the smart button will display
+   :guilabel:`Next Meeting` with the date of the upcoming meeting.
+
+The calendar view loads, showing the currently scheduled meetings and events. The meetings and
+events displayed are for the employees who are listed under the :guilabel:`Attendees` section on the
+right side of the calendar view. To change the currently loaded meetings and events being displayed,
+uncheck the person who's calendar events are to be hidden. Only the checked attendees will be
+visible on the calendar.
```

Comment:
No- BUT there should be no reason someone unassociated with recruitment/interviews/etc is creating a meeting in the DB - at east that's what I was thinking. Id never go in and schedule an interview for our recruitment team, you know? So you can add people to the meeting, I just honestly can't imagine a scenario where another person was scheduling this- since recruitment is typically more confidential. You know? I am not sure if that answers your comment fully but I think it does. I didn't add the search since that seemed to be... how should I say this- common knowledge on how it works since it's all over the DB? I always struggle with just how granular to go- because I end up with docs this long  ^_^ But in the end I thought this was an understood chunk.